dev/setup_db.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-DELETE_ALL_DOCUMENTS=true \
-npm --prefix=`dirname "$0"`/../data-serving/scripts/setup-db/ run-script setup-cases
-npm --prefix=`dirname "$0"`/../data-serving/scripts/setup-db/ run-script setup-restricted-cases
-npm --prefix=`dirname "$0"`/../data-serving/data-service/ run-script import-data
\ No newline at end of file
+CONN="${CONN:-mongodb://localhost/covid19}"
```

Comment:
Is this only for local testing? Then I would set CONN directly to prevent errors if CONN is somehow set manually and dev/setup_db.sh is run by mistake. Alternatively disallowing mongodb+srv in CONN could help prevent this as well.

Reply:
OK, that's a good point.